scripts/crawl.sh

```diff
@@ -16,6 +16,6 @@ test -f ${TOUCHFILE} && (echo "Another crawling process is still running (${TOUC
 # We are in the protected section
 touch ${TOUCHFILE}
 
-cd ${BASEDIR}
+cd ${BASEDIR} && git pull --no-edit main master
```

Comment:
Have you considered doing that in the Python script instead?

Reply:
Not sure if that is a good idea just in case `scripts/crawl.py` gets changed by the pull since if I do pull in `scripts/crawl.py` then any modification in `scripts/crawl.py` won't be seen/applied until the next run of the crawler